Dockerfile

```diff
@@ -5,5 +5,5 @@ RUN pip install --no-cache-dir --upgrade -r /code/requirements/docker.txt
 RUN pip install -e .
 RUN pip install opentelemetry-distro==0.38b0 opentelemetry-exporter-otlp-proto-grpc==1.17.0
 
-CMD ["opentelemetry-instrument", "uvicorn", "dj.api.main:app", "--host", "0.0.0.0", "--port", "8000"]
+CMD ["opentelemetry-instrument", "uvicorn", "dj.api.main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
```

Comment:
The reason I removed this `--reload` is because it interferes with the opentelemetry auto instrumentation. The server still start up fine but in dj-demo none of the traces will make it to the jaeger container. That being said, it is definitely convenient to have the reload on while developing. Maybe we could do something like this:
```docker
ARG RELOAD="--reload"
ENV RELOAD ${RELOAD}
CMD ["opentelemetry-instrument", "uvicorn", "dj.api.main:app", "--host", "0.0.0.0", "--port", "8000", "${RELOAD}"]
```
Then in the [docker-compose.yml](https://github.com/DataJunction/dj-demo/blob/main/docker-compose.yml#L26) file for dj-demo we can do:
```yaml
services:
  dj:
    ...
    build: 
        context: "${CORE_SERVICE}"
        args:
            RELOAD: ""
 ...
```
Definitely not urgent since the tracing is just a "nice to have" part of the demo. Feel free to disregard this as a non-blocker and we can worry about solving this later.

Reply:
Good point! I'll make this change